src/components/accordion/Accordion.spec.tsx

```diff
@@ -226,7 +226,14 @@ describe('<Accordion>', () => {
 
     expect(item.getAttribute('aria-expanded')).toEqual('false');
     expect(accordion.queryByRole('region')).toBeNull();
+
+    // This one does not trigger onFocus method in React 17
     accordion.getByRole('button').focus();
+
+    // This one does not set document.activeElement
+    // https://github.com/jsdom/jsdom/issues/2586
+    fireEvent.focus(accordion.getByRole('button'));
```

Comment:
what do you think we can do about this?

Reply:
No idea, I was not able to fix this differently.
The component works correctly though, it collapes on Enter/Space when motion is reduced:

https://github.com/brainly/style-guide/assets/52164548/7cb61829-9804-478b-be32-f66ddb8e82ae